tasks/vm.py

```diff
@@ -88,3 +95,25 @@ def _get_os_family(os_family: Optional[str]) -> str:
             f"The os family '{os_family}' is not supported. Possibles values are {', '.join(os_families)}"
         )
     return os_family
+
+def _get_os_information(os_family: Optional[str], ami_id: Optional[str]) -> tuple:
```

Comment:
Isn't that what the Go code already does?

Reply:
Here we have at least an opportunity to guess the os family (rather than defaulting to ubuntu) but maybe it can be overridden later in the go code. Don't hesitate to tell me if I should move things elsewhere.
Plus I guess currently `arch` is forced to amd64